po/cs.po

```diff
@@ -0,0 +1,139 @@
+# Czech translation for flatseal package.
+# Copyright (C) 2020 THE flatseal'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the flatseal package.
+#
+# Daniel Rusek <mail@asciiwolf.com>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: flatseal\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-02-03 17:02+0100\n"
+"PO-Revision-Date: 2020-02-03 14:58+0100\n"
+"Last-Translator: Daniel Rusek <mail@asciiwolf.com>\n"
+"Language-Team: none\n"
+"Language: cs\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.2.4\n"
```

Comment:
Where's plurals ? Let me know if you need help running these ninja commands.

Reply:
I updated the potfile/pofiles using `ninja flatseal-pot`. I am not sure why plurals were not added.

Reply:
Maybe because the file already exists? Try "removing" it and run it again?

Reply:
I have tried, still nothing. I have at least force pushed the other fixes.

Reply:
Sorry, I don't have enough free time left (and enough know-how) to look into this. :-/

Reply:
`"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"`

There you go! No hurry though.

Reply:
Thanks!